lib/tasks/documentation.rake

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+task "documentation" do
```

Comment:
Oh I just realised this is in core, shouldn't it be in `plugins/chat/lib/tasks/documentation.rake` ? If that's not possible fine, just wondering

Reply:
The idea is to provide a general solution for plugins and core to generate documentation

Reply:
No probs :+1: